### PR TITLE
Added docopt cpp dependency to zimtools

### PR DESCRIPTION
**Pull request by MiguelRocha**
This PR adds the docopt cpp library to the build environment. It will be needed for zimdump.

### CODE REVIEW

File: kiwixbuild/dependencies/__init__.py [hunk 1]
```diff
@@ -26,5 +26,6 @@
     zim_tools,
     zimwriterfs,
     zlib,
-    zstd
+    zstd,
+    docoptcpp
```
Comment (mgautierfr) [inline kiwixbuild/dependencies/__init__.py:30]: Please add it by alphabetical order.

File: kiwixbuild/versions.py [hunk 2]
```diff
@@ -48,6 +48,7 @@
   'zlib' : '1.2.8',
   'lzma' : '5.2.4',
   'zstd' : '1.4.4',
+  'docoptcpp' : '0.6.2',
```
Comment (mgautierfr) [inline kiwixbuild/versions.py:51]: You also need to increment the `base_deps_meta_versions` just above.

File: kiwixbuild/dependencies/all_dependencies.py [hunk 3]
```diff
@@ -16,5 +16,9 @@ def get_dependencies(cls, platformInfo, allDeps):
             # zimwriterfs
             if platformInfo.build not in ('android', 'win32'):
                 base_deps += ['libmagic', 'gumbo']
+            
+            # zimtools
+            if platformInfo.build not in ('android','ios'):
```
Comment (mgautierfr) [inline kiwixbuild/dependencies/all_dependencies.py:21]: This is `iOS` not `ios`.

**Comment by MiguelRocha** [comment]
@mgautierfr I removed the compilation of docoptcpp from windows and iOS as we talked. Let me know if everything is ok.

**Comment by mgautierfr** [comment]
We are good !
I have just squash the commit of the Pr as the intermediate steps are useless here.

**Pull request merged**